src/clinical/clinical-entities.ts

```diff
@@ -1,4 +1,5 @@
 import { DeepReadonly } from 'deep-freeze';
+import { CoreClinicalEntites } from '../submission/submission-entities';
```

Comment:
clinical entities should not reference submission entities, this `CoreClinicalEntites` a clinical property not submission property